doc/sphinx/requirements.txt

```diff
@@ -6,3 +6,6 @@ ase
 sphinx==7.2.6
 furo==2023.9.10
 myst-parser
+pydantic
+toml
+PyQt5
```

Comment:
Is PyQT5 now a part of the full installation automatically?

Reply:
No, this is installed just by the automated system that builds the HTML pages at ReadTheDocs. It's because during the build it imports the `GUIWidgets` module which in turn imports `PyQt5`, which was producing a warning.